src/core/thread/mle.cpp

```diff
@@ -3869,6 +3869,64 @@ void Mle::HandleLinkProbe(const Message &aMessage, const Ip6::MessageInfo &aMess
 }
 #endif // OPENTHREAD_CONFIG_MLE_LINK_METRICS_ENABLE
 
+#if OPENTHREAD_CONFIG_MAC_SSED_TO_SSED_LINK_ENABLE
+void Mle::HandleSsedBeacon(const Mac::RxFrame &aFrame)
+{
+    otError           error    = OT_ERROR_NONE;
+    Child *           peerSsed = nullptr;
+    Mac::Address      macDest;
+    Mac::Address      macSource;
+    Mac::PanId        destPan;
+    const uint8_t *   cur = aFrame.GetHeaderIe(Mac::Frame::kHeaderIeCsl);
+    const Mac::CslIe *csl;
+
+    VerifyOrExit(cur != nullptr, error = OT_ERROR_PARSE);
+    SuccessOrExit(error = aFrame.GetSrcAddr(macSource));
+    SuccessOrExit(error = aFrame.GetDstAddr(macDest));
+    SuccessOrExit(error = aFrame.GetDstPanId(destPan));
+
+    VerifyOrExit(mAttachState == kAttachStateIdle, error = OT_ERROR_INVALID_STATE);
+
+    // Skip none broadcast null data
+    VerifyOrExit(macDest.IsBroadcast());
+
+    // Skip null data with short or none source address
+    VerifyOrExit(macSource.IsExtended());
+
+    // Skip unmatched dest PAN ID
+    VerifyOrExit(Get<Mac::Mac>().GetPanId() == destPan);
+
+    peerSsed = Get<ChildTable>().FindChild(macSource.GetExtended(), Child::kInStateAnyExceptInvalid);
+
+    if (peerSsed == nullptr)
+    {
+        VerifyOrExit((peerSsed = Get<ChildTable>().GetNewChild()) != nullptr, error = OT_ERROR_NO_BUFS);
```

Comment:
We should add a TODO here, for S2S end device, there is only one entry in its Child Table.